build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+echo "Starting webpack build..."
+cd /Users/balajialwar/Berkeley/repos/nbgitpuller-link-generator-webextension
```

Comment:
This contains a path specific to your device.

Reply:
Removed that file as it is not needed in the repo (?)